api/app/Extensions/JWTAuth/UserClaim.php

```diff
@@ -0,0 +1,24 @@
+<?php namespace App\Extensions\JWTAuth;
+
+use Tymon\JWTAuth\Claims\Claim;
+
+class UserClaim extends Claim
+{
+    /**
+     * The claim name.
+     *
+     * @var string
+     */
+    protected $name = '_user';
+
+    /**
+     * Validate the user claim.
+     *
+     * @param  mixed  $value
+     * @return boolean
+     */
+    protected function validate($value)
+    {
+        return is_array($value) || is_null($value);
```

Comment:
This should also not accept a null value. If there is no user, the token is not valid.